synapse/tests/utils.py

```diff
@@ -922,11 +962,11 @@ def testfunc():
 
         self.raises(exc, testfunc)
 
-    def eq(self, x, y):
+    def eq(self, x, y, mesg=None):
```

Comment:
This is nice to have anyway (let's you log an additional message if a test fails)